src/manifest/mod.rs

```diff
@@ -36,6 +40,63 @@ struct CargoPackage {
     repository: Option<String>,
 }
 
+#[derive(Default, Deserialize)]
+struct CargoMetadata {
+    #[serde(rename = "wasm-pack")]
```

Comment:
I think `serde(default)` may need to be here and all lower fields as well, right? Currently if you have `[package.metadata.foo]` that may fail to deserialize as well as just `[package.metadata.wasm-pack.profile.dev]` (and not other profiles).

`package.metadata.wasm-pack.profile.dev` is also a bit of a mouthful, so perhaps we can drop the `profile` name as well?

Reply:
I guess it comes down to whether we will ever have non profile related things here. It seems probable to me that we will (eg the JS env target) and so I don't think we should get rid of the `profile` path segment.